Framework/PythonInterface/plugins/algorithms/MagnetismReflectometryReduction.py

```diff
@@ -417,7 +417,9 @@ def calculate_scattering_angle(self, ws_event_data):
 
         #theta = (dangle - dangle0) * math.pi / 360.0
         theta = (dangle - dangle0) * math.pi / 180.0 / 2.0 + ((direct_beam_pix - ref_pix) * pixel_width) / (2.0 * det_distance)
-
+        if theta < 0:
+            logger.warning("The calculated scattering angle is negative: taking absolute value")
+            theta = -theta
```

Comment:
The code would be clearer if you change it to either `theta = -1. * theta` or `theta = abs(theta)`.